setup-earthly/libs/setup-earthly.js

```diff
@@ -0,0 +1,36 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const child_process = require('child_process');
+
+const IS_WINDOWS = process.platform === "win32";
+const IS_MAC = process.platform === "darwin";
+
+try {
+  console.log(`Setting up earthly`);
+
+  install_cmd = "sudo /bin/sh -c 'wget https://github.com/earthly/earthly/releases/latest/download/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly'"
```

Comment:
Might be worth following the airbnb style guide https://github.com/airbnb/javascript which is popular with ES6+. I believe JS prefers cammel-case for variables.